### PR TITLE
Show relative date tooltips on configuration pages

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

medic/medic-webapp#3835

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: templates/partials/configuration_upgrade.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <dt translate>instance.upgrade.deployed_by</dt>
         <dd>{{deployInfo.user}}</dd>
         <dt translate>instance.upgrade.at</dt>
-        <dd>{{deployInfo.timestamp | date}}</dd>
+        <dd ng-bind-html="deployInfo.timestamp | relativeDate"></dd>
```
Comment (SCdF) [inline templates/partials/configuration_upgrade.html]: This is not supposed to be relative

File: templates/partials/configuration_upgrade.html [hunk 2]
```diff
@@ -35,7 +35,7 @@
       <ul ng-show="versions.releases.length" class="table table-striped">
         <li ng-repeat="release in versions.releases" class="row">
           <div class="col-xs-5"><code>{{release.id}}</code></div>
-          <div class="col-xs-4" title="{{release.value.build_time}}">{{release.value.build_time | date}}</div>
+          <div class="col-xs-4" ng-bind-html="release.value.build_time | relativeDate"></div>
```
Comment (SCdF) [inline templates/partials/configuration_upgrade.html]: again, not supposed to be relative

File: templates/partials/configuration_upgrade.html [hunk 3]
```diff
@@ -48,7 +48,7 @@
         <ul ng-show="versions.betas.length" class="table table-striped">
           <li ng-repeat="release in versions.betas" class="row">
             <div class="col-xs-5"><code>{{release.id}}</code></div>
-            <div class="col-xs-4" title="{{release.value.build_time}}">{{release.value.build_time | date}}</div>
+            <div class="col-xs-4" ng-bind-html="release.value.build_time | relativeDate"></div>
```
Comment (SCdF) [inline templates/partials/configuration_upgrade.html]: Like wise, not supposed to be relative

File: static/js/controllers/inbox.js [hunk 4]
```diff
@@ -520,7 +520,7 @@ var feedback = require('../modules/feedback'),
             .tooltip({
               placement: 'bottom',
               trigger: 'manual',
-              container: $(this).closest('.inbox-items, .item-content')
+              container: $(this).closest('.inbox-items, .item-content, .page')
```
Comment (SCdF) [inline static/js/controllers/inbox.js]: Also, was this what actually fixed it? That we had some tooltip logic, and it needs one of these classes to bind it's hidden html to?

Comment (garethbowen) [reply static/js/controllers/inbox.js]: Yes this is what fixed it. You have to opt in to bootstrap tooltips. If you don't specify the `container` then it defaults to `body` which doesn't work in our UI which floats around all over the place.

**Pull request merged**